Modules/Module_Processing/Module_Population/xASL_stat_GetROIstatistics.m

```diff
@@ -633,7 +635,7 @@
 				pvPrimary = ones(size(DataIm)); % no masking
 				bSkipPVC = 1;
             else
-                switch x.S.TissueMaskingLocal{1}
+                switch x.S.TissueMaskingLocal

```

Comment:
If you change this, you should also change this in xASL_module_Population. We don't need this as cell still because different atlases can have different TissueMaskingLocal? E.g. an atlas can contain GM and WM regions. But we can also keep that as a potential future feature.

Reply:
In xASL_module_Population - this is always a string and never a cell. That's why this had to be fixed here...